packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -199,7 +209,7 @@ class DatePickerInput extends React.Component {
           placeholder={placeholder}
           onFocus={onFocus}
           onBlur={onBlur}
-          aria-label={intl.formatMessage({ id: 'Terra.datePicker.date' })}
```

Comment:
This is a text input. A date value is being read by screen readers as text, not as a date. For example, "01/01/2020" is read as "zero one slash zero one slash twenty twenty" instead of "January 1st twenty twenty". The original attempt was to set `aria-hidden=true` on this input to prevent it from reading the text and then use visually-hidden-text to just announce the string as a date. 

Since this input is a focusable element, we cannot use `aria-hidden=true` or `role=none`. https://www.w3.org/TR/using-aria/#4thrule. We don't know of other ways to prevent AT from reading the value as text. Anyone know a way to do that?

For now, we are just allowing it to read both the value as text followed by date.